elastic/integrations
# Make cockroachDB GA

- Enhancement


## What does this PR do?

This PR make CockroachDB release GA

## Checklist



## How to test this PR locally

Run the cockroachDB instance.
Add the integration and metrics should start flowing through Kibana.

## Related issues



- Closes https://github.com/elastic/integrations/issues/3550
- Closes https://github.com/elastic/integrations/issues/480

## Screenshots
